component-models.json

```diff
@@ -665,38 +665,21 @@
     "id": "columns",
     "fields": [
       {
-        "component": "multiselect",
+        "component": "select",
         "valueType": "string",
         "name": "classes",
         "value": "",
-        "label": "Columns Style",
+        "label": "Content Style",
         "options": [
           {
-            "name": "Content alignment",
-            "children": [
-              {
-                "name": "Content centered",
-                "value": "content-center"
-              },
-              {
-                "name": "Content left aligned",
-                "value": "content-left"
-              }
-            ]
+            "name": "Info Grid, centered",
+            "value": "info-grid-centered"
           },
           {
-            "name": "Clickable Images",
-            "children": [
-              {
-                "name": "Yes",
-                "value": "clickable-images"
-              },
-              {
-                "name": "No",
-                "value": ""
-              }
-            ]
```

Comment:
@msagolj , this has to be kept, this will handle the case where columns has images which are clickable as a whole (reference: social media icons in footer)
please keep this as is.

Reply:
please see: https://github.com/aemsites/piramal/issues/21

Reply:
> @msagolj , this has to be kept, this will handle the case where columns has images which are clickable as a whole (reference: social media icons in footer) please keep this as is.

I see. In that case wouldn't it be better to simply add a 3rd option in the  select of this PR e.g. 
'Social Media Icons' ?

if we add this option back in a multi-select we can also set this clickable image option for other types of columns , where it will not work or makes no sense, so it would be confusing for the author.

Reply:
sure, that can work too
but i wont name it as 'social media icons' as the same functionality might be required somewhere else too.
so 'clickable-images' would still be better (by default should be off, if on the class should appear on that div to handle the click)

Reply:
as discussed we turn it back into a multiselect with:
![image](https://github.com/user-attachments/assets/b6440951-7ed0-461c-a004-c76509d07ea9)